cvem/connectors/one/OpenNebula.py

```diff
@@ -35,7 +35,7 @@ class DISK(XMLObject):
 		values = ['CLONE','READONLY','SAVE','SOURCE','TARGET' ]
 
 class TEMPLATE(XMLObject):
-		values = [ 'CPU', 'MEMORY', 'NAME', 'RANK', 'REQUIREMENTS', 'VMID', 'VCPU', 'PACKEDMEMORY', 'MAXMEMORY' ]
+		values = [ 'CPU', 'MEMORY', 'NAME', 'RANK', 'REQUIREMENTS', 'VMID', 'VCPU' ]
```

Comment:
As PACKEDMEMORY and MAXMEMORY are located in the TEMPLATE (at least in versions <= 4.12) is better to maintain these items in this list to make it work in both versions.